$NetBSD: patch-CMakeLists.txt,v 1.5 2025/09/23 22:38:44 gdt Exp $

Omit a warning that is unrecognized on gcc 7.

Sent upstream by email.

--- CMakeLists.txt.orig	2025-09-23 22:12:59.828352847 +0000
+++ CMakeLists.txt
@@ -72,7 +72,6 @@ if("${CMAKE_C_COMPILER_ID}" STREQUAL "GN
     # -Wold-style-cast
     -Woverloaded-virtual
     -Wzero-as-null-pointer-constant
-    -Wdeprecated-copy-dtor
   )
 elseif("${CMAKE_C_COMPILER_ID}" MATCHES "Clang")
   set(PROJ_common_WARN_FLAGS ${PROJ_common_WARN_FLAGS}
